grails-app/views/picklist/_page.gsp

```diff
@@ -1,4 +1,17 @@
 <%@ page import="org.apache.commons.lang.StringEscapeUtils" %>
+<%
+    def comparator = { a,b ->
```

Comment:
I'd prefer not to use scriplets in the GSP if possible, so let's move this to the PicklistItem or a utility class? The easiest way to do this is to implement Comparable and add a compareTo() method. Check out Requisition and RequisitionItem for an example.

Reply:
Oh sorry this was for RequisitionItem, not PicklistItem.

Reply:
We can leave this for now, but it would be better to move the comparator into its own class and/or implement multiple Comparators in the RequisitionItem class and then add sortRequsitionItemsByWhatever1() and sortRequisitionItemsByWhatever2() to Requisition or even a utility class.

https://stackoverflow.com/questions/4432774/how-do-i-make-2-comparable-methods-in-only-one-class

Reply:
We can leave it. I'll try to determine the best way to handle this and do it some other time.